metadata-ingestion/examples/mce_files/dynamodb.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
Should the folder be renamed `mcp_files`?

Reply:
this is datahub out of box directory. we can move our dynamodb file to a separator folder if we'd like to.